packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec

```diff
@@ -54,6 +54,7 @@ BuildRequires: rubygems-devel
 BuildRequires: ruby
 BuildArch: noarch
 Provides: rubygem(%{gem_name}) = %{version}
+Provides: deprecated()
```

Comment:
What does this do? :D

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/deprecating-packages/#_marking_a_package_deprecated

AFAIU, deprecates the package? I mean, this package should be installed instead of rubygem. I guess it's enough to have `Obsoletes` in the new package, but just to be sure I've marked rubygem one as deprecated.

Reply:
TIL

Yeah I don't think it does anything valuable for us. The `Obsoletes` should be sufficient.

Reply:
Dropped the commit with deprecation.